pkg/reloader/reloader.go

```diff
@@ -69,6 +68,8 @@ import (
 	"sync"
 	"time"
 
+	"github.com/minio/sha256-simd"
```

Comment:
Same grouping issue

Reply:
oops. I think a go fmt extension did that.